packages/theme/src/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const LIGHT = 'light'
+export const DARK = 'dark'
+
+export enum THEME {
```

Comment:
Couldn't it be const? Const enums has much simpler inner structure

https://www.typescriptlang.org/play?ts=4.4.4#code/KYOwrgtgBAogjFA3gKCmqAbAlgcwBYAuANKugCYCGATgNbIC+yyAxgPYgDOBUwCAvLDgA6bPgJM2nbqEiwATElJpRhEuiiVaDCey48FAmHKGaaQA

Reply:
I'm unsure about swc and const enum, can you please research on that?